diff --git a/node_modules/@dnd-kit/dom/index.js b/node_modules/@dnd-kit/dom/index.js
--- a/node_modules/@dnd-kit/dom/index.js
+++ b/node_modules/@dnd-kit/dom/index.js
@@ -2012,7 +2012,7 @@
     controller.signal.onabort = () => this.handleCancel(event);
     controller.onEvent(event);
     this.controller = controller;
-    const documents = getDocuments();
+    const documents = getDocuments(getDocument(source.element));
     const unbindListeners = this.listeners.bind(documents, [
       {
         type: "pointermove",
diff --git a/node_modules/@dnd-kit/dom/index.cjs b/node_modules/@dnd-kit/dom/index.cjs
--- a/node_modules/@dnd-kit/dom/index.cjs
+++ b/node_modules/@dnd-kit/dom/index.cjs
@@ -2013,7 +2013,7 @@
     controller.signal.onabort = () => this.handleCancel(event);
     controller.onEvent(event);
     this.controller = controller;
-    const documents = utilities.getDocuments();
+    const documents = utilities.getDocuments(utilities.getDocument(source.element));
     const unbindListeners = this.listeners.bind(documents, [
       {
         type: "pointermove",
